schema_based_on_diagram.sql

```diff
@@ -0,0 +1,123 @@
+-- Table: public.treatments
+
+-- DROP TABLE IF EXISTS public.treatments;
+
+CREATE TABLE IF NOT EXISTS public.treatments
+(
+    id integer NOT NULL,
+    type character varying COLLATE pg_catalog."default",
+    name character varying COLLATE pg_catalog."default",
+    CONSTRAINT treatments_pkey PRIMARY KEY (id)
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public.treatments
+    OWNER to postgres;
+CREATE TABLE IF NOT EXISTS public.patients
+(
+    id integer NOT NULL,
+    name character varying COLLATE pg_catalog."default" NOT NULL,
+    date_of_birth date NOT NULL,
+    CONSTRAINT patients_pkey PRIMARY KEY (id)
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public.patients
+    OWNER to postgres;
+	
+CREATE TABLE IF NOT EXISTS public.medical_histories
+(
```

Comment:
- [ ] As requested by the previous reviewer, Kindly make sure all  "join table" for the many-to-many relationships is created, even if they are not shown on the diagram. Therefore you should create a table between `medical_histories` and `treatments` (name can be different):

  ```sql
    CREATE TABLE medical_histories_has_treatments (
      medical_history_id int refrences medical_histories(id),
      treatment_id int refrences treatments(id),
      );
  
  ```
* Kindly address this 👍🏻